Content/concepts/advanced/runwith.html

```diff
@@ -0,0 +1,107 @@
+	
+	<h2 id="sql-plus-integration-via-runWith-for-liquibase-pro-users">SQL Plus integration via <code>runWith</code> for Liquibase Pro users</h2>
+		<p>For Liquibase Pro users, SQL Plus integration is built-in.</p>
+		<ol>
+			<li>Make sure you have the latest version of Liquibase (download Liquibase 3.10.0 or higher).
+			<li>Have SQL Plus in your PATH (or configure it in the accompanying <code>liquibase.sqlplus.conf</code> file)
+			<li>Add <code>runWith=sqlplus</code> attribute to a changeset
+		</ol>
+
+<h2 id="runwith-example">runWith Examples:</h2>
+	<p><strong>Formatted SQL changeset using runWith attribute</strong></p>
+		<code>
+		--liquibase formatted sql<br>
+		<br>
+		--changeset myauthorname:2314 runwith=sqlplus<br>
+		DECLARE l_emp_name VARCHAR2(250);<br>
+		l_emp_no NUMBER;<br>
+		l_salary NUMBER;<br>
+		l_manager VARCHAR2(250);<br>
+		BEGIN<br>
+		INSERT INTO emp(emp_name,emp_no,salary,manager) VALUES('BBB',1000,25000,'AAA');<br>
+		...[and so on]...<br>
+		END;<br>
+		/
+		</code>
+	
+	<p><strong>XML changelog with inline SQL changeset using runWith attribute</strong></p>
+		<figure class="highlight"><pre xml:space="preserve"><code class="language-xml" data-lang="xml"><span class="nt">&lt;changeSet</span>  <span class="na">id=</span><span class="s">"2"</span>  <span class="na">author=</span><span class="s">"myauthorname"</span>  <span class="na">runWith=</span><span class="s">"sqlplus"</span><span class="nt">&gt;</span>  
+	<span class="nt">&lt;sql&gt;</span>
+		DECLARE l_emp_name VARCHAR2(250);
+		l_emp_no NUMBER;
+		l_salary NUMBER;
+		l_manager VARCHAR2(250);
+		BEGIN
+		INSERT INTO emp(emp_name,emp_no,salary,manager) VALUES('BBB',1000,25000,'AAA');
+		...[and so on]...
+		END;
+		/
+	<span class="nt">&lt;/sql&gt;</span>  
+	
+<span class="nt">&lt;/changeSet&gt;</span></code></pre>
+</figure>
+
+
+
+	<p><strong>XML/YML/JSON changelog pointing to a sql file with raw sql in it</strong></p>
+		<figure class="highlight"><pre xml:space="preserve"><code class="language-xml" data-lang="xml"><span class="nt">&lt;changeSet</span>  <span class="na">id=</span><span class="s">"2315"</span>  <span class="na">author=</span><span class="s">"myauthorname"</span>  <span class="na">runWith=</span><span class="s">"sqlplus"</span><span class="nt">&gt;</span>  
+	<span class="nt">&lt;sqlFile</span>
+	path="my/path/file.sql"
+	...etc etc... &gt;</span>  
+<span class="nt">&lt;/changeSet&gt;</span></code></pre>
+</figure>
+		
+
+
+
+<h2 id="the-optional-conf-file">The optional .conf file</h2>
+<p>For Liquibase Pro users, there is a <code>liquibase.sqlplus.conf</code> file which lives alongside your liquibase.properties or POM file, where you can optionally specify some useful key-value pairs for configuring your executor, such as:<br>
```

Comment:
It should just be alongside the liquibase.properties file, not by the POM file.

Reply:
ok, i ll tweak that